stdlib/2and3/_dummy_threading.pyi

```diff
@@ -0,0 +1,198 @@
+# Stubs for threading
```

Comment:
No need for this comment

Reply:
Should I remove it from `threading.pyi`? As they're going to be checked consistent